pyproject.toml

```diff
@@ -8,7 +8,7 @@ dependencies = [
     "arelle-release>=2.25,<2.28",
     "catalystcoop.pudl @ git+https://github.com/catalyst-cooperative/pudl.git",
     "coloredlogs>=14",
-    "dask<2024.6",  # Temporary pin -- conda packaging problem upstream
+    "dask>=2024",
```

Comment:
This temporary problem went away, and I needed to change the hash of pyproject.toml to clear the environment cache, which had the bad version of a package stuck in it.